src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -107,15 +104,29 @@ private void buildDWithinQuery(CriteriaBuilder builder, CriteriaQuery query, Roo
      */
     @SuppressWarnings("unchecked")
     private void buildSearchQuery(CriteriaBuilder builder, CriteriaQuery query, Root root, String searchQuery,
-                                  String latitude, String longitude, String distance, Boolean orderByResults) {
+                                  Double latitude, Double longitude, Integer distance, Boolean orderByResults) {
         // TODO: Query for tags
         // TODO: Filters
-        if (searchQuery != null) {
-            this.buildSimilarityQuery(builder, query, root, searchQuery, 0.1, orderByResults);
+
+        List<Predicate> predicates = new ArrayList<>();
+
+        if (searchQuery != null && !searchQuery.trim().equals("")) {
```

Comment:
You should be able to use `!StringHelper.isNullOrEmpty(searchQuery)`

Reply:
Done